api/docs/release.dox

```diff
@@ -136,6 +136,7 @@ clients.
 The changes between version \DR_VERSION and 7.1.0 include the following minor
 compatibility changes:
 
+ - Added the type struct dr_zmm_t. dr_zmm_t is not used or supported yet.
```

Comment:
Yes it's not used yes. The reason for that I put it in the release notes was that it does show up in exported types. Since it will change again to *being* used soon, I guess it is safe to not mention it at all. I removed the note, making the "#' comment obsolete.